draft-ietf-quic-manageability.md

```diff
@@ -1018,13 +1019,12 @@ heuristics or machine learning instead of processing observed header
 information.
 
 However, it is questionable whether connection migrations must be supported
-during a DDoS attack. However, for unintentional mitigations, as is usually the
-case for NAT rebindings, the connection ID can be used to track
-connections (see {{sec-flow-association}}). If the connection
-migration is not visible to the network
-that performs the DDoS detection, an active, migrated QUIC connection may be
-blocked by such a system under attack. As soon as the connection blocking is
-detected by the client, the client may rely on the fast resumption mechanism
+during a DDoS attack. While unintended migration where the connection ID
+stays unchanged can be more easiliy supported, it might be acceptable that
+connection migrations of active QUIC connections that are not visible to
+the network that performs the DDoS detection may be blocked by such a system
+under attack. As soon as the connection blocking is detected by the client,
+the client may be able to rely on the fast resumption mechanism
```

Comment:
```suggestion
during a DDoS attack. While unintended migration without a connection ID
change can be more easily supported, it might be acceptable to not
support migrations of active QUIC connections that are not visible to
the network functions performing the DDoS detection. 
As soon as the connection blocking is detected by the client,
the client may be able to rely on the fast resumption mechanism
```